tests-integration/fixtures/docker-compose.yml

```diff
@@ -1,14 +1,14 @@
 services:
```

Comment:
If we are using Docker Compose (not Helm chart), most of the images (etcd/zookeeper/minio/kafka) can use official images. It can reduce the request limit for the same public AWS repo.

Reply:
I added the [pull-test-deps-images.sh](https://github.com/GreptimeTeam/greptimedb/pull/6847/files#diff-ce55861c53f324649aa83eb8b76d4598bd8a4793a82c25483a7c5791c6086cdf) to work around. However, it's better to use a github cache.